### PR TITLE
Feature/percentage of affected people

### PR DESCRIPTION
**Pull request by ShraddhaJ11**

### CODE REVIEW

**Comment by anamritraj** [comment]
This looks great!! 🚀

File: src/components/Stats.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 
+const calculatePercentage = (cases, total) => {
+  return ((cases / total) * 100).toFixed(2).toLocaleString('en');
```
Comment (kovoorprajwal) [inline src/components/Stats.js:5]: A small comment which doesn't matter for functionality wise. You can remove the brackets if you want as it's a single line function.

**Comment by kovoorprajwal** [comment]
Requested review is optional as it's a non-functional change request.

**Comment by ShraddhaJ11** [comment]
Thanks for the review, I will remove these brackets in next PR.

**Pull request merged**